src/pages/case-study/copenhagen-atomics.tsx

```diff
@@ -0,0 +1,365 @@
+          <Image
+            alt="Copenhagen atomics develops Generation IV reactors, which are
+        molten salt reactors."
+            description={
+              <>
+                Copenhagen atomics develops Generation IV reactors, which are
+                molten salt reactors.&nbsp;
+                <a
+                  href="https://www.copenhagenatomics.com/"
+                  target="_blank"
+                  rel="noreferrer noopener"
+                >
+                  Image source
+                </a>
+                .
+              </>
+            }
+            src="/img/pages/case-study/copenhagen-atomics/illustration.png"
+            width={777}
+            height={350}
+          />
+          <p className="font-size--large">
+            Modelling on how offshore windmill energy achieved economies of
+            scale through mass production, Copenhagen Atomics aims to scale up
+            the production of the molten salt reactors to provide Energy as a
+            Service. Depending on the energy needs of a client, the company will
+            lease the reactors and sell the generated energy while managing the
+            equipment after decommission.
+          </p>
+          <p className="font-size--large">
+            In this case study, Lasse Tarp, the Software Group Manager from
+            Copenhagen Atomics, introduces us to the company&#39;s software
+            architectural design with QuestDB at its core.
+          </p>
+
+          <h3>Architecture overview</h3>
+
+          <p className="font-size--large">
+            We deploy our architecture on Azure and host API servers as well as
+            QuestDB&#39;s engine on Kubernetes. Each reactor sends up to 200
+            sensor data points to the API servers. Data is collected every 10
+            milliseconds. The number of reactor sensor values is likely to grow
+            from 200 to near 1000 in the future.
+          </p>
+
+          <p className="font-size--large">
+            From the API servers we have two parallel processes: ingesting
+            sensor data to QuestDB using the .NET ILP client and sending queried
+            data to our plot systems for monitoring. We customize our own plot
+            systems for fetching and displaying data.
+          </p>
+          <Image
+            src="/img/pages/case-study/copenhagen-atomics/iot-monitors-graph.png"
+            width={828}
+            height={331}
+            description="An example screenshot of a plot system that monitors IoT devices"
+          />
+
+          <p className="font-size--large">
+            At Copenhagen Atomics we also produce and sell loops for testing
+            equipment that are not produced by us.{" "}
+            <a href="https://www.copenhagenatomics.com/technology">
+              https://www.copenhagenatomics.com/technology
+            </a>
+            . The plot shown above is a selection of some of the temperature
```

Comment:
```suggestion
              <a href="https://www.copenhagenatomics.com/technology">
                equipment
              </a>
            that are not produced by us. The plot shown above is a selection of some of the temperature
```
Forgot to indicate which text to add the link. I think the link should go to "equipment" and the above is my attempt to add it...